website/src/pages/Home/Home.jsx

```diff
@@ -83,24 +146,43 @@ const Home = () => {
             </button>
           </div>
         </div>
-        <img src={logo} alt="logo" className="home__top__logo" />
+        <img src={logo} alt="logo" className="home_top_logo" />
       </article>
 
       <article className="home__body">
-        <section className="home__body__how__it__works">
+        <section className="home_bodyhowit_works">
           <h1>How it works</h1>
-          <div className="how__it__works__cards__container">
+          <div className="how_itworkscards_container">
             {howItWorksItems.map((item, key) => (
-              <div className="how__it__works__card" key={key}>
+              <div className="how_itworks_card" key={key}>
                 <h3>{item.title}</h3>
                 <p>{item.content}</p>
               </div>
             ))}
           </div>
         </section>
+
+        <section className="micro_and_macro">
+          <h1 className="home_bodysection_title">
+            Micro and Macro Benchmarks
+          </h1>
+
+          <div className="micro_andmacro_cards">
+            {microMacroContent.map((item, key) => (
+              <div key={key} className="micro_andmacro_card">
+                <h3>{item.title}</h3>
+                <ul>
+                  {microMacroContent[key].points.map((point, i) => (
+                    <li><h5>{point.title}</h5> <p>{point.content}</p></li>
```

Comment:
<img width="233" alt="Capture d’écran 2023-08-18 à 13 39 38" src="https://github.com/vitessio/arewefastyet/assets/116645689/c909bfcb-5291-4d5a-baba-4c3684879488">

A key is missing in this part of the code because I'm getting this error.

Reply:
Oh yes, Fixed it